rosdoc2/verbs/build/inspect_package_for_settings.py

```diff
@@ -49,7 +49,7 @@
     ## be assumed for 'sphinx-apidoc' invocation. The user can provide the path
     ## (relative to the 'package.xml' file) where the Python modules defined by this
     ## package are located.
-    python_source: '{package_name}'
+    # python_source: '{package_name}'
```

Comment:
Can the above cases not be covered by setting this non-standard location in their config files? 

Implementing custom discovery logic seem fragile and like it will end up with a lot of maintenance in the longer term versus setting the non-standard location in the configuration for those handful of packages.  And we can make sure that we provide a useful error/warning such that developers can know to set this setting in their config file.

Reply:
Just to be clear, the reason for the change in the line that you quoted is because {package_name} is the default. I would expect that a sample config file shows what the default is, but does not set it unless setting it is required. It is not.